notebooks/Tracking_drift_check.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<p>Same here. I would remove this unless it is critical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>